package/MDAnalysis/transformations/translate.py

```diff
@@ -107,6 +104,9 @@ def center_in_box(ag, center='geometry', point=None, pbc=None):
     """
     
     pbc_arg = pbc
+    point = np.array(point)
+    if point.any() and len(point)!=3:
```

Comment:
if point is `[0, 0, 0]`, then `point.any()` will be False. Also, I would recommend avoiding `len` on arrays: I got weird behaviours at times. Anyway, the logic you had before where `point` could be an array or `None` was misplaced but safer.